src/styles/_layout.scss

```diff
@@ -1,19 +1,31 @@
 .container {
-    width: 100%;
-    margin-left: auto;
-    margin-right: auto;
-    padding-left: $spacing-md;
-    padding-right: $spacing-md;
-    
-    @include respond-to('sm') {
-      max-width: map-get($breakpoints, 'sm');
-    }
-    
-    @include respond-to('md') {
-      max-width: map-get($breakpoints, 'md');
-    }
-    
-    @include respond-to('lg') {
-      max-width: map-get($breakpoints, 'lg');
-    }
-  }
\ No newline at end of file
+  width: 100%;
+  margin-left: auto;
+  margin-right: auto;
+  padding-left: $spacing-md;
+  padding-right: $spacing-md;
+
+  @include respond-to("sm") {
+    max-width: map-get($breakpoints, "sm");
+  }
+
+  @include respond-to("md") {
+    max-width: map-get($breakpoints, "md");
+  }
+
+  @include respond-to("lg") {
+    max-width: map-get($breakpoints, "lg");
+  }
+}
+*,
+body,
+html {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
```

Comment:
This is where you should use the font family you want to use. If this is not the one you are using then write the one you have in your designs and remove it from the css code. if this is the one you want to use then keep this as is and remove from css code

Reply:
fixed font-family for each component